lib/compile/csdl2openapi.js

```diff
@@ -2019,6 +2032,7 @@ see [Expand](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-prot
 
         if (suffix === SUFFIX.read && type["@ODM.root"]) schemas[schemaName]["x-sap-root-entity"] = type["@ODM.root"]
```

Comment:
Do we bind the ER annotations mandatory to ODM usage? Can we decouple that? A developer might only adopt ER annotations and not ODM.

Reply:
ok, understood from discussion, that they are decoupled.